.gitmodules

```diff
@@ -0,0 +1,4 @@
+
+[submodule "deps/bls-signatures"]
+	path = deps/bls-signatures
+	url = https://github.com/filecoin-project/bls-signatures.git
```

Comment:
can we fix commit somehow, so that when bls-signatures updated our implementation does not crash

Reply:
It's fixed in the "deps/bls-signatures" file: `Submodule bls-signatures added at 92e4c4` (commit 92e4c48c12bf575191903018f3bfc4a53fc73023)